yazi-plugin/preset/plugins/extract.lua

```diff
@@ -36,6 +36,16 @@ function M:entry(args)
 	end
 end
 
+function M:is_tar_compatible(url)
+	local tar_extensions = { ".tar", ".tgz", ".tar.gz", ".tar.bz2" }
```

Comment:
Are you sure `tar -xvf` also works for `.tar.gz` and `.tar.bz2` files?

Reply:
Yeah, it did work. But think it's still necessary to change to `-a` to make it  `--auto-compress` to increase the compatibility.